src/zepto.js

```diff
@@ -437,9 +437,10 @@ var Zepto = (function() {
 
     ready: function(callback){
       // don't use "interactive" on IE <= 10 (it can fired premature)
+      // pipe any ready callback call over task/event queue (#1235)
       if (document.readyState === "complete" ||
           (document.readyState !== "loading" && !document.documentElement.doScroll))
-        callback($)
+        setTimeout(function(){ callback($) })
```

Comment:
Good find! I guess we should be compatible with jQuery. Can you add an explicit `0` for the 2nd argument to setTimeout?